runtime/src/bank.rs

```diff
@@ -2349,6 +2351,34 @@ impl Bank {
         });
     }
 
+    pub fn update_last_restart_slot(&self) {
+        let feature_flag = self
+            .feature_set
+            .is_active(&feature_set::last_restart_slot_sysvar::id());
+
+        if feature_flag {
+            let last_restart_slot = {
+                let slot = self.slot;
+                let hard_forks = self.hard_forks();
+                let hard_forks_r = hard_forks.read().unwrap();
+
+                // Only consider hard forks <= this bank's slot to avoid prematurely
+                hard_forks_r
```

Comment:
Looks like only half of the comment made it in

Reply:
Ahh yes silly me the second line did not `// applying a hard fork that is set to occur in the future.` make sense to me just reading it standalone, it makes more sense combining with the first line.

Reply:
I will fix it ASAP.